floqast_sftp/app.py

```diff
@@ -0,0 +1,273 @@
+import csv
+import io
+import json
+import logging
+from datetime import date, datetime
+
+import boto3
+import paramiko
+
+import requests
+
+logging.basicConfig(level=logging.INFO)
+
+ssm_client = None
+
+_csv_url = "https://mips-api.finops.sageit.org/balances?show_inactive_codes"
```

Comment:
what do you think about passing in this var as a lambda parameter in case it ever changes?

Reply:
my initial thinking was that it's just one line and easy to update if we need to. but it's probably worth it to make it a parameter so that if we do need to update it, we can do it from org-formation instead of needing to release a new lambda version.